scu-it/src/fiskaltrust.Middleware.SCU.IT.EpsonRTPrinter/EpsonRTPrinterSCU.cs

```diff
@@ -594,16 +593,32 @@ private async Task<ReceiptResponse> PerformDailyCosing(ReceiptResponse receiptRe
         }
     }
 
-    private async Task<(List<SignaturItem> signaturItems, string ftStateData)> PerformZeroReceiptOperationAsync()
+    private async Task<ProcessResponse> PerformZeroReceiptOperationAsync(ReceiptRequest request, ReceiptResponse receiptResponse)
     {
         await ResetPrinter();
         var result = await QueryPrinterStatusAsync();
         var signatures = SignatureFactory.CreateZeroReceiptSignatures().ToList();
+        if (request.IsXReportZeroReceipt())
+        {
+            var fiscalReport = new FiscalReport
+            {
+                XReport = new XReport()
+            };
+            var response = await SendRequestAsync(SoapSerializer.Serialize(fiscalReport));
+            using var responseContent = await response.Content.ReadAsStreamAsync();
+            var reportResponse = SoapSerializer.DeserializeToSoapEnvelope<ReportResponse>(responseContent);
+            if (!result?.Success ?? false)
```

Comment:
```suggestion
            if (!(result?.Success ?? false))
```
Just makes it more clear what's going on with out having to think about operator precedence ^^